### PR TITLE
29-created new header

### PR DESCRIPTION
**Pull request by firity**
created new header component...i still have to do some changes

### CODE REVIEW

File: client/components/header-component.js [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-  <a class="link" href="/pages/members/members.html">members</a>
-  <a class="link" href="/pages/about-us/about-us.html">about us</a>
-  <a class="link" href="/pages/contact-us/contact-us.html">contact us</a>
-  <button>Log In</button>
-</div>`
+  template: `<section class="header">
+  <div class="header-container">
+      <div class="header-left">
+      <a href="https:" class="logo">
+      <img src="/client/public/assets/logo_pc_full.png" alt="logo" width="100" height="80"/></a>
```
Comment (yoshimalaise) [inline client/components/header-component.js:6]: Normally when running the server using nom run start the url path should not contain "/client" as the client is the only folder that is served by the backend

**Review by yoshimalaise — CHANGES_REQUESTED** [review]

File: client/components/header-component.js (hunk 1 above)
Comment (dirk-jacobs) [reply client/components/header-component.js:6]: exactly, because of this, the logo is not longer rendered

File: client/styles/styles.css [hunk 2]
```diff
@@ -9,6 +9,74 @@ body {
...
+}
+
+.header a {
+float: left;
+text-align: center;
+padding: 12px;
+text-decoration: none;
+font-size: 24px;
+margin-right: 25px;
+font-family: 'Montserrat medium';
```
Comment (dirk-jacobs) [inline client/styles/styles.css:32]: Missing sans-serif which gives something very different


According to Google

image

File: client/styles/styles.css [hunk 3]
```diff
@@ -9,6 +9,74 @@ body {
   color: #2F195F;
 }
 
+.header {
```
Comment (dirk-jacobs) [inline client/styles/styles.css:12]: Should be better aligned with design, please check the desig in figma

- height should be 90px
- no top padding
- do we need % padding ?
- should be better aligned in the center 
- Sign in button too small
- Text in sign-in button 24px iso 18px

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
Still room for improvement

File: client/styles/styles.css (hunk 3 above)
Comment (firity) [reply client/styles/styles.css:12]: thank you for the review

**Pull request closed without merging**